### ISSUE TITLE
Details of labelling 1,000 wikiHow articles as ordered or not for whether their steps are ordered

### ISSUE THREAD

**Issue by WangRuoke** [zharry29/wikihow-goal-step#3]
You have mentioned the labelling of articles for the step ordering task. 
> We ask 3 annotators to label 1,000 wikiHow articles as ordered or not as a coarse-grained approximation for whether their steps are ordered. We finetune a pre-trained RoBERTa model using 5- fold cross-validation, finding an average precision of 88%. We then ask a 4th annotator to label another 40 articles as the held-out test set, where the finetuned model achieves 100% precision. Finally, we only consider articles that the model predicts as ordered (around 40%) for the Step Ordering task.

I would like to know more about the details of this part. Did you give one article only one label or use other methods? As some articles have multiple methods and not all steps in these methods have temporal relations, like this article [https://www.wikihow.com/Buy-Treasury-Bonds-in-Australia#Buying-Australian-Government-Bonds]. Please share how to deal with this kind of articles.

Thx!

**Comment by veronica320** [comment]
Corresponded via email. Marked as closed.
